src/data/showcases.js

```diff
@@ -1015,6 +1023,15 @@ const Showcases = [
     source: null,
     tags: [ "pooltool", "analytics"],
   },
+  {
+    title: "Lido Nation",
+    description:
+      "Series of experiments which seeks to generate the highest levels of community innovation.",
```

Comment:
I see it more as a aggregation of of existing catalyst proposals, results, feedbacks, entities, etc (as against any experiments that were innovated themselves), but do not feel too strongly about the description

Reply:
How about 'Series of existing catalyst proposals, results, feedbacks and many more'

What do you think?

Reply:
Ye - sounds better (series or aggregation - either)